chrome-go/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM golang:1.17.4
+
+RUN wget -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
```

We should[ try to limit the number of RUN commands](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers), each RUN command creates a new layer, the echo into the file probably doesn't warrant one maybe we merge them into one item?
```suggestion
RUN wget -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
  && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list
```

---

Yeah good point, updated